components/org.wso2.carbon.identity.api.server.userstore/org.wso2.carbon.identity.api.server.userstore.v1/src/main/java/org/wso2/carbon/identity/api/server/userstore/v1/core/ServerUserStoreService.java

```diff
@@ -102,6 +102,8 @@ public void deleteUserStore(String userstoreDomainId) {
             UserStoreConfigService userStoreConfigService = UserStoreConfigServiceHolder.
                     getUserStoreConfigService();
             userStoreConfigService.deleteUserStore(base64URLDecodeId(userstoreDomainId));
+        } catch (IdentityUserStoreClientException ignored) {
+
```

Comment:
Shall log this?

Reply:
Also, Can we say that the deletion was a success if an exception occurred.

Reply:
Added a debug log. Here we are caching a client exception. Since this is a delete operation the only input given by the user is an identifier. So the only client exception that can occur here is the resource not being available.